semantic-conventions/src/tests/data/yaml/errors/examples/example_bool.yaml

```diff
@@ -9,4 +9,4 @@ groups:
         type: boolean
         required: always
         brief: 'test'
-        examples: [true, false, yes]
+        examples: [true, false, true]
```

Comment:
Those examples don't really make sense. You could instead check that `yes` and `1` are rejected as expected.

Reply:
I added a new test. It is possible that this test was supposed to work differently, since it is in the errors/ directory, but the accompanying test does not assert on anything.

Reply:
You are right, the test was wrong, and the YAML was intentionally invalid. Fixed in 1272f57, 
e205225